views/tables/Table/TableMenu.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Button, Menu, MenuItem, MenuList, Typography } from '@mui/material'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+
+const MENU_ITEM_STYLES = { display: 'flex', gap: '32px' }
+const COLOR_STYLES = { color: 'black.A100' }
+const TableMenu = (props) => {
+  const { menuItems } = props
+  const [menuAnchor, setMenuAnchor] = React.useState(null)
+  const open = Boolean(menuAnchor)
+  const onClose = () => setMenuAnchor(null)
+
+  return (
+    <>
+      <Button
```

Comment:
I wonder if we should use the IconButton for buttons that use Icons..?
https://mui.com/material-ui/api/icon-button/

Reply:
yeah we can do that - https://github.com/AMP-SCZ/dpdash/pull/494/commits/11517e6bfe6e3dc6e8a140beced61d2f18fbeb42